interfaces/cython/cantera/_onedim.pyx

```diff
@@ -1559,10 +1566,13 @@ cdef class Sim1D:
         >>> s.restore(filename='save.yaml', name='energy_off')
 
         .. versionchanged:: 3.0
-            Implemented return value for meta data
+            Implemented return value for meta data; loglevel is no longer supported
```

Comment:
Does it make sense to add a second version changed directive here?

Reply:
Since both were changed in the same version, it should be ok to combine?

Reply:
I was thinking the second directive would make the existence of both changes more prominent, as opposed to putting the fairly unrelated things in the same HTML box. Just a thought though ☺️

Reply:
The way it renders on the docs, I believe having a single directive per version is preferable?

Reply:
Alright, thanks for checking!